netsim/daemons/bird.yml

```diff
@@ -20,6 +20,8 @@ virtualbox:
 features:
   bgp:
     activate_af: true
+    ipv6_lla: true
```

Comment:
BGP over IPv6-LLA does not work. IPv4 AF over GUA IPv6 EBGP session (RFC 8950) works